test/e2e/grpc_test.go

```diff
@@ -35,21 +37,34 @@ import (
 
 type grpcTest func(*testing.T, *test.ResourceObjects, *test.Clients, string, string)
 
+// getPort gets port from a URL
+func getPort(u string) (int, error) {
```

Comment:
Why even get the port here if you're not interested in it at all? Should it be `hasPort` and return a boolean?

Reply:
good idea, done